pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -59,16 +58,16 @@ theme: responsive_full_width
             %p.no-margin-bottom=hoc_s(:ai_intro_list_desc_03)
     .clear
 
--# Hiding this section s of 7/7/23
--# %section.bg-primary-light.blog-post-callout{id: "ai-pl-callout"}
--#   .wrapper
--#     %i{class: "#{icon_megaphone}"}
--#     .text-wrapper
--#       %h2.heading-lg=hoc_s(:ai_pl_callout_title)
--#       %p
--#         %strong=hoc_s(:ai_pl_callout_desc_01)
--#       %p.body-three=hoc_s(:ai_pl_callout_desc_02)
--#       %a.link-button{href: "https://code.zoom.us/webinar/register/8616874889915/WN_TGKpfOD8Rw-z5Q-D3ZiuLQ", target: "_blank", rel: "noopener noreferrer"}=hoc_s(:signup_submit_label)
+
+%section.bg-primary-light.blog-post-callout{id: "ai-pl-callout"}
+  .wrapper
+    %i{class: "#{icon_megaphone}"}
+    .text-wrapper
+      %h2.heading-lg=hoc_s(:ai_pl_callout_title)
+      %p
+        %strong=hoc_s(:ai_pl_callout_desc_01)
+      %p.body-three=hoc_s(:ai_pl_callout_desc_02)
+      %a.link-button{href: "/ai/pl/101", target: "_blank", rel: "noopener noreferrer"}=hoc_s(:signup_submit_explore)
```

Comment:
Since this page is on our site you can remove the `target: "_blank", rel: "noopener noreferrer"` here.